renderdoc/driver/vulkan/vk_layer.cpp

```diff
@@ -163,7 +163,7 @@ VkResult VKAPI_CALL hooked_vkCreateInstance(const VkInstanceCreateInfo *pCreateI
                                             const VkAllocationCallbacks *pAllocator,
                                             VkInstance *pInstance)
 {
-  WrappedVulkan *core = new WrappedVulkan("");
+  WrappedVulkan *core = new WrappedVulkan(0);
```

Comment:
triggers an assert/error around renderdoc/serialise/serlialise.cpp:659 (Serialiser::Serialiser()) as path would be a valid string and fopen fails to open "" (empty string).
Replacing it with a nullptr causes the implementation to use a dummy log (renderdoc/driver/vulkan/vk_core.cpp:277, WrappedVulkan::WrappedVulkan())